### ISSUE TITLE
Brutallus Script + Crash (BB #443)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#443]
This issue was migrated from bitbucket.
**Original Reporter:** sh1fty88
**Original Date:** 17.03.2013 17:59:11 GMT+0000
**Original Priority:** critical
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443

<hr>

Script for madrigosa & Brutallus is missing/incorrect.

When killing Brutallus, the server crashes randomly.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** sh1fty88
**Original Date:** 17.03.2013 17:59:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-3435613

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Vicher
**Original Date:** 18.03.2013 05:32:13 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-3439106

<hr>

Go to UnitAI.cpp line 187 and change

```
#!c++

while (num > targetList.size())
```

to

```
#!c++

while (num < targetList.size())
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** sh1fty88
**Original Date:** 18.03.2013 19:20:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-3446850

<hr>

Thanks! solved!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 28.12.2013 05:52:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7513784

<hr>

This should be submitted as a commit, if the issue has been resolved.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 29.12.2013 02:02:39 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7519455

<hr>

No more developers present on OC at the moment.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 29.12.2013 03:31:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7519645

<hr>

Due to the official web site going offline, is this the only channel of communication the OregonCore currently uses?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 29.12.2013 07:30:12 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7520274

<hr>

I believe so. And if you're looking for developers. We had one but he went off to another project. So noone has power to commit etc. anymore.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 29.12.2013 08:57:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7520554

<hr>

I just hope this project will not be abandoned, and that there are still people here that are willing to continue the development.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.12.2013 18:37:48 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7533168

<hr>

I agree :)

Oregon has been my first, and will always be my only core

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 01.01.2014 22:46:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/443/#comment-7545974

<hr>

Resolved issue #443 thanks to Vicher

→ <<cset ab5ad943d547>>
